zebra/zebra_evpn.h

```diff
@@ -98,6 +98,9 @@ struct zebra_evpn_t_ {
 	/* Corresponding VxLAN interface. */
 	struct interface *vxlan_if;
 
+	/* Corresponding SVI interface. */
+	struct interface *svi_if;
```

Comment:
This will also need to be re-orged later. When vlan-aware-bundle comes in a BD will be represented by a version of zebra_evpn_access_bd.